docs/tutorials/control/flutter-app.md

```diff
@@ -0,0 +1,732 @@
+
+   4. Replace its contents with the following:
+
+   ```yaml {class="line-numbers linkable-line-numbers"}
+   include: package:flutter_lints/flutter.yaml
+
+   linter:
+     rules:
+       avoid_print: false
+       prefer_const_constructors_in_immutables: false
+       prefer_const_constructors: false
+       prefer_const_literals_to_create_immutables: false
+       prefer_final_fields: false
+       unnecessary_breaks: true
+       use_key_in_widget_constructors: false
+   ```
+
+### Configure iOS-specific code
+
+Now you'll update some configurations in the iOS-specific code to support the [Viam Flutter SDK](https://flutter.viam.dev/).
+
+1. Open <file>ios/Podfile</file>.
+   If Podfile does not exist in that directory, you can generate it by running `flutter pub get` in the root directory of your app.
+
+   At the top of the file you will see the following lines:
+
+   ```{class="line-numbers linkable-line-numbers"}
+   # Uncomment this line to define a global platform for your project
+   # platform :ios, '11.0'
+   ```
+
+   Change that to this:
+
+   ```{class="line-numbers linkable-line-numbers"}
+   # Uncomment this line to define a global platform for your project
+   platform :ios, '13.0'
+   ```
+
+2. Open <file>ios/Runner/Info.plist</file>.
+   It will look something like this:
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The info.plist file open in VS Code. The second line includes 'doctype plist public' and an Apple URL. The fourth line is a dict tag. The fifth line is a key, and subsequent lines contain keys, strings and arrays.">}}
+
+3. Insert this code into the first line after the `<dict>`:
+
+   ```{class="line-numbers linkable-line-numbers"}
+     <key>NSLocalNetworkUsageDescription</key>
+     <string>Smart Machine App requires access to your device's local network to connect to your devices.</string>
+     <key>NSBonjourServices</key>
+     <array>
+         <string>_rpc._tcp</string>
+     </array>
+   ```
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist-insert.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The info.plist file open in VS Code. An arrow labeled 'insert here' indicates that you should insert the code right after the fourth line in the file.">}}
+
+   It should look like this:
+
+   {{<imgproc src="/tutorials/flutter-app/info-plist-pasted.png" resize="1200x" style="max-width:900px" declared-dimensions="true" alt="The resulting file. After the non-indented <dict> tag on the fourth line, the pasted key, string, key, and array (containing an indented string) are indented inside the dict tag, spanning lines five through ten. The pasted content is at the same level of indentation as the rest of the keys and strings below it.">}}
+
+### Create the main file
+
+1. Open the <file>main.dart</file> file under the <file>lib/</file> directory.
```

Comment:
supernit: Previously we used this kinda notation:

```suggestion
1. Open the <file>lib/main.dart</file> file.
```

No real preference, just a vote for consistency, esp since IDE left-nav shows folder structure so prominently.

Reply:
can do